src/main/java/rx/internal/util/ObjectPool.java

```diff
@@ -55,6 +55,7 @@ private ObjectPool(final int min, final int max, final long validationInterval)
         initialize(min);
 
         schedulerWorker = Schedulers.computation().createWorker();
+        // FIXME should subscription returned be added to some composite
```

Comment:
There is no place to unsubscribe this worker: it will exist until the JVM quits.